webpack.config.js

```diff
@@ -8,10 +9,20 @@ module.exports = {
 			( plugin ) =>
 				plugin.constructor.name !== 'DependencyExtractionWebpackPlugin'
 		),
-		new DependencyExtractionWebpackPlugin(),
+		new DependencyExtractionWebpackPlugin( {
+			injectPolyfill: true,
+		} ),
 	],
+	resolve: {
+		extensions: [ '.ts', '.tsx', '.json', '.js', '.jsx' ],
```

Comment:
It doesn't look like this project uses Typescript, so some of these extensions might not be necessary
```suggestion
		extensions: [ '.json', '.js', '.jsx' ],
```